packages/ts/react-crud/src/property-info.ts

```diff
@@ -48,6 +59,12 @@ function determinePropertyType(model: AbstractModel): PropertyType {
   } else if (constructor === BooleanModel) {
     return 'boolean';
   }
+
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+  if ((model as EnumModel)[_enum]) {
+    return 'enum';
+  }
```

Comment:
We can use `instanceof` checks here, as well as the checks above:
```ts
  // Otherwise detect by model instance
  if (model instanceof StringModel) {
    return 'string';
  } else if (model instanceof NumberModel) {
    return 'decimal';
  } else if (model instanceof BooleanModel) {
    return 'boolean';
  } else if (model instanceof EnumModel) {
    return 'enum';
  }
```